### PR TITLE
[v0.32] Doc/2337 - Connector Tutorial rework

### PR DESCRIPTION
**Pull request by thomasyit**
**A description of the changes proposed in the pull request**:

 - breakup the task4 into two topics, Hummingbot configuration, and, testing and debugging.
- add testing reference for each task to the testing and debugging topics
- highlight binance as an example for the devs
- fixed broken links for the connectors

Screenshot 2020-10-01 at 1 30 54 PM
Screenshot 2020-10-01 at 1 30 42 PM
Screenshot 2020-10-01 at 1 30 34 PM

### CODE REVIEW

**Review by dennisocana — CHANGES_REQUESTED** [review]
LGTM overall and resolve conflicting files. Just a minor note when contributing to docs:

1. Use `development` branch as base instead of `master` to avoid conflicts
2. Prefix each commit as indicatd in the contributing guide https://github.com/CoinAlpha/hummingbot/blob/master/CONTRIBUTING.md#make-commits-to-your-feature-branch

File: documentation/docs/developers/connectors/index.md [hunk 1]
```diff
@@ -10,18 +10,21 @@ There are [existing connectors](https://github.com/CoinAlpha/hummingbot/tree/mas
...
 
-- Centralized exchange: 
-    - [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/binance) 
-    - [Coinbase Pro](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/coin-base_pro) - [Huobi](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/huobi)
-    - [Bittrex](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bittrex)
-    - Ethereum DEX (0x open order book): [Radar Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/radar_relay)
-    - Ethereum DEX (0x open order book w/coordinator support): [Bamboo Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/market/bamboo_relay)
+The following list some examples/templates that you can refer to when building the connector:
+
+- [Binance](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/binance)
```
Comment (Nullably) [inline documentation/docs/developers/connectors/index.md:19]: Crypto.com connector should actually be the first example here as it is mainly Python implementation (others are mainly Cython) and will be easier for other developers to follow.

**Review by Nullably — CHANGES_REQUESTED** [review]

File: documentation/docs/developers/connectors/index.md (hunk 1 above)
Comment (thomasyit) [reply documentation/docs/developers/connectors/index.md:19]: Screenshot 2020-10-05 at 9 18 38 AM
Noted and changed.

File: documentation/docs/developers/connectors/index.md [hunk 2]
```diff
@@ -10,18 +10,22 @@ There are [existing connectors](https://github.com/CoinAlpha/hummingbot/tree/mas
...
+- [Bittrex](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/bittrex)
+- Ethereum DEX (0x open order book): [Radar Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/radar_relay)
+- Ethereum DEX (0x open order book w/coordinator support): [Bamboo Relay](https://github.com/CoinAlpha/hummingbot/tree/master/hummingbot/connector/exchange/radar_relay)
 
 ## Exchange connector requirements
-1. A complete set of exchange connector files as listed [above](#exchange-connector-files).
+1. A set of exchange connector files as listed [above](#exchange-connector-files).
 2. Unit tests (see [existing unit tests](https://github.com/CoinAlpha/hummingbot/tree/master/test/integration)):
 
     1. Exchange market test ([example](https://github.com/CoinAlpha/hummingbot/blob/master/test/integration/test_binance_market.py))
```
Comment (Nullably) [inline documentation/docs/developers/connectors/index.md:31]: Can you change this to `test/connector/exchange/crypto_com/test_crypto_com_exchange.py`?

Comment (thomasyit) [reply documentation/docs/developers/connectors/index.md]: edit the testing links to  crypto.com directory

**Review by Nullably — APPROVED** [review]
LGTM

**Review by dennisocana — APPROVED** [review]
LGTM

**Pull request merged**